src/main/java/org/jboss/tools/intellij/openshift/actions/component/DebugComponentAction.java

```diff
@@ -191,7 +186,7 @@ private boolean checkOdoDebugRunning(Odo odo, String projectName, String applica
             try {
                 // find an available port and use it
                 ServerSocket serverSocket = new ServerSocket(0);
-                port = serverSocket.getLocalPort();
+                port = Integer.valueOf(serverSocket.getLocalPort());
```

Comment:
I don't see the rationale for this change ?

Reply:
IntelliJ quick fix proposal. reverted.